lib/MusicBrainz/Server/Controller/Admin/Attributes.pm

```diff
@@ -93,6 +93,8 @@ sub edit : Chained('attribute_base') Args(1) RequireAuth(account_admin) {
     my $form = $c->form( form => $form_name, init_object => $attr );
 
     if ($c->form_posted && $form->process( params => $c->req->params )) {
+        $form->field("entity_type")->value( $attr->entity_type );
```

Comment:
I removed it in https://github.com/metabrainz/musicbrainz-server/pull/471/commits/cde0df93294647929c713687922ce70b55df3d01 (where field `entity_type` input is enabled just before form submission) since it is unnecessary anyway, unless we want proper validation which should then take place in a separate package for attributes with `entity_type` field.